### PR TITLE
Set default max_fee in account functions

### LINKED ISSUE #261: Account.send does not have a default value for max_fee
The [documentation for `send`](https://github.com/OpenZeppelin/nile#send) states that max_fee should default to 0.
However, [Account.`send` always requires max_fee](https://github.com/OpenZeppelin/nile/blob/main/src/nile/core/account.py#L131). Instead, it should set the default to `max_fee=None` (which it did before, but was changed in [this PR](https://github.com/OpenZeppelin/nile/commit/8fc00121fcb93c85ea78b541ae106e6b69d24ce2#diff-8ec97b3a1c3f7fc3e205808742ef74f16501c22b45d31bd3481ef9087b815ac4)).

This applies to all of these functions in Account:
- `send`
- `simulate`
- `estimate_fee`

### PR DESCRIPTION
**Pull request by ericglau**
Fixes #261

### CODE REVIEW

File: tests/commands/test_account.py [hunk 1]
```diff
@@ -170,15 +170,21 @@ def test_send_sign_transaction_and_execute():
         )
 
 
+def test_send_defaults():
+    account = Account(KEY, NETWORK)
+
+    account.send(account.address, "method", [1, 2, 3])
```
Comment (andrew-fleming) [inline tests/commands/test_account.py:176]: Just to be thorough, I'd create a mock for `account.send` and call the mock's `assert_called_once_with` to assert the passed default values.

**Review by andrew-fleming — COMMENTED** [review]
@ericglau good improvement! I left a small comment :)

**Review by martriay — DISMISSED** [review]

**Review by ericnordelo — APPROVED** [review]
LGTM!

**Review by andrew-fleming — APPROVED** [review]
Looks good to go!

**Review by martriay — APPROVED** [review]

**Pull request merged**